pkg/migrations/op_set_unique.go

```diff
@@ -25,12 +26,43 @@ func (o *OpSetUnique) Start(ctx context.Context, conn db.DB, latestSchema string
 	table := s.GetTable(o.Table)
 	column := table.GetColumn(o.Column)
 
-	// Add a unique index to the new column
-	if err := addUniqueIndex(ctx, conn, table.Name, column.Name, o.Name); err != nil {
-		return nil, fmt.Errorf("failed to add unique index: %w", err)
+	for retryCount := 5; retryCount > 0; retryCount-- {
```

Comment:
This format is more idiomatic:

```suggestion
	for retryCount := 0; retryCount < 5; retryCount++ {
```